discord/channel.py

```diff
@@ -3056,7 +3061,11 @@ def _update_group(self, data: GroupChannelPayload) -> None:
         self.owner_id: Optional[int] = utils._get_as_snowflake(data, 'owner_id')
         self._icon: Optional[str] = data.get('icon')
         self.name: Optional[str] = data.get('name')
-        self.recipients: List[User] = [self._state.store_user(u) for u in data.get('recipients', [])]
+        self.recipients: List[User] = []
+
+        recipients = data.get('recipients')
+        if recipients is not None:
+            self.recipients = [self._state.store_user(u) for u in recipients]
```

Comment:
Undo this, the old impl achieved the same result.

```suggestion
        self.recipients: List[User] = [self._state.store_user(u) for u in data.get('recipients', [])]
```